.eslintrc.js

```diff
@@ -18,10 +18,11 @@ module.exports = {
   },
   settings: {
     react: {
-      version: 'detect',
+      version: '17.0',
```

Comment:
Need to lock this because, by default, we do not actually install React anymore — `@remote-ui/mini-react` is used instead. Locking this avoids a warning from the ESLint plugin.